docs/elements/jira/tips.md

```diff
@@ -0,0 +1,20 @@
+---
+heading: Jira
+seo: Tips | Jira | Cloud Elements API Docs
+title: Tips
+description: Jira Tips
+layout: sidebarelementdoc
+breadcrumbs: /docs/elements.html
+elementId: 1
+parent: Back to Element Guides
+order: 75
+---
+
+# Tips
+
+The Tips section includes more information about the element including FAQs, endpoint limitations, and general tips.
+
+
+## General
+
+* Although Jira supports the Delete /users API, there is not enough information returned in the event to identify the associated instance; therefore, events for DELETE /users has not been implemented.
```

Comment:
JIRA, and add tics around Delete /users.

A couple of things with the therefore clause. Events is plural, which would take a have instead of a has. But it kind of isn't plural.... Maybe try:
"therefore we do not support `DELETE /users` events."